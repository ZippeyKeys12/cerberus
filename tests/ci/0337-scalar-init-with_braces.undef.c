// this is undefined (ยง6.7.9#11)
int xs[] = {{10, 20}};
