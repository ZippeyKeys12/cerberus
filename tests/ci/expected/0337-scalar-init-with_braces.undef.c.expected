Undefined {ub: "UB081_scalar_initializer_not_single_expression", stderr: "", loc: "<2:13--2:21>"}
